google-cloud-trace/lib/google/cloud/trace/faraday_middleware.rb

```diff
@@ -20,11 +20,36 @@ module Google
   module Cloud
     module Trace
       class FaradayMiddleware < Faraday::Middleware
+        ##
+        # # Trace FaradayMiddleware
+        #
+        # A faraday middleware that setup request/response labels for trace.
+        #
+        # ## Installing
+        #
+        # To use this middleware, simply install it in your middleware stack.
+        # Here is an example configuration enable the Trace middleware:
+        #
+        # ```ruby
+        # conn = Faraday.new(:url => 'http://sushi.com') do |faraday|
+        #   # enable cross project tracing with option to true
+        #   faraday.use Google::Cloud::Trace, enable_cross_project_tracing: true
+        #   faraday.request  :url_encoded             # form-encode POST params
+        #   faraday.response :logger                  # log requests to $stdout
+        #   faraday.adapter  Faraday.default_adapter  # make requests with Net::HTTP
+        # end
+        # ```
+
+        def initialize(app, options: {})
+          super(app)
+          @enable_cross_project_tracing = options.delete(:enable_cross_project_tracing) || false
```

Comment:
Probably better to avoid modifying options argument passed in. Consider:
```ruby
@enable_cross_project_tracing = options.fetch :enable_cross_project_tracing, false
```